src/VisualStudio/Core/Def/Implementation/LanguageClient/VisualStudioOOPConnectionLanguageClient.cs

```diff
@@ -21,20 +20,21 @@
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.ServiceHub.Client;
 using Microsoft.VisualStudio.LanguageServer.Client;
-using Microsoft.VisualStudio.LanguageServices.Remote;
 using Microsoft.VisualStudio.Threading;
 using Microsoft.VisualStudio.Utilities;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageClient
 {
+    /// <summary>
+    /// VS implementation of <see cref="ILanguageClient"/> that works by remoting the requests over to our OOP server to
+    /// actually perform the work.
+    /// </summary>
     [ContentType(ContentTypeNames.CSharpContentType)]
     [ContentType(ContentTypeNames.VisualBasicContentType)]
     [Export(typeof(ILanguageClient))]
-    [Export(typeof(LanguageServerClient))]
-    internal sealed class LanguageServerClient : ILanguageClient
+    [Export(typeof(VisualStudioOOPConnectionLanguageClient))]
+    internal sealed class VisualStudioOOPConnectionLanguageClient : ILanguageClient
```

Comment:
i don't love thi sname.  But i wanted to convey that it's on hte VS side, and supplies the connection to the oop server.

Reply:
What does "on the VS side" mean? Wouldn't this just be `OOPLanguageServer`? Can this not be used by VS for Mac, for example?

Reply:
"on the VS side" means this code is executing in devenv.exe.  `OOPLanguageServer` makes me think it's running in ServiceHub.Roslyn....exe.  I'm trying to convey that this is hte devent piece that *connects* to the piece running in OOP.

Reply:
That makes a lot of sense 👍